gallery/src/components/NxTabs/NxTabsPage.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import {
+  NxTable,
+  NxTableBody,
+  NxTableCell,
+  NxTableHead,
+  NxTableRow
+} from '@sonatype/react-shared-components';
+
+import NxTabsSimpleExample from './NxTabsSimpleExample';
+
+const NxTabsSimpleExampleCode = require('!!raw-loader!./NxTabsSimpleExample').default;
+
+export default function NxTabsPage() {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <p className="nx-p">
+          A set of accessible tabs components which must be used together.
+        </p>
+
+        <section className="nx-tile--subsection">
+          <h3 className="nx-h3">NxTabs</h3>
```

Comment:
```suggestion
        <section className="nx-tile-subsection">
          <header className="nx-tile-subsection__header">
            <h3 className="nx-h3">NxTabs</h3>
          </header>
```
And similar changes to the others.

Reply:
3eef0e0

Reply:
> 3eef0e0

That didn't get the `nx-tile-subsection` class name quite right

Reply:
80ce379

Reply:
> 80ce379

That's still wrong.  Please see the `nx-tile` example.  If the styling isn't coming through like that example, you don't have the classes right.

Reply:
181e194663f5960dbaf5794a9488b59030a2088c